blocks/hero/product-hero.css

```diff
@@ -113,7 +113,8 @@
 }
 
 .block.hero.cuvitru .link .button-container a {
-  color: var(--cuvitru);
+  background-color: var(--cuvitru);
```

Comment:
@bstopp, I didn't see any other buttons/links that would be affected by this in the hero and thought this was ok due to this code identifying the Cuvitru variant of the hero block.  If you see a downside to this that I missed, please let me know.